README.md

```diff
@@ -60,26 +60,15 @@ To make changes to Tilt, read the [developer guide](DEVELOPING.md).
 For bugs and feature requests, file an [issue](https://github.com/windmilleng/tilt/issues)
 or check out the [feature roadmap](ROADMAP.md).
 
-## Privacy
+## Telemetry and Privacy
+We're a small company trying to make Tilt awesomer. We can do this better if we understand which features people are using and which bugs people are running into. Tilt can send telemetry data to https://events.windmill.build if you opt-in. You can do this in the web UI or by running `tilt analytics opt in`. It really helps us improve your experience.
```

Comment:
```suggestion
We're a small company trying to make Tilt awesomer. We can do this better if we understand which features people are using and which bugs people are running into. Tilt can send telemetry data to https://events.windmill.build if you opt in. You can do this in the web UI or by running `tilt analytics opt in`. It really helps us improve your experience.
```
Nit: `opt in` is the verb.

Reply:
Isn't this using it as a verb?

Reply:
Right, I'm saying if you're using it as a verb it should not have the hyphen (which I removed in my suggestion above). (This is pure pedantry but it seems more reasonable in user-facing documentation 🤷‍♀ )